internal/transport/handler_server.go

```diff
@@ -361,7 +362,9 @@ func (ht *serverHandlerTransport) HandleStreams(startStream func(*Stream), trace
 		for buf := make([]byte, readSize); ; {
```

Comment:
Can the TODO above be removed now?

Reply:
No, this doesn't implement buffer sharing for handler server.